front/src/App.js

```diff
@@ -127,13 +120,10 @@ class App extends Component {
   }
 
   onConnectionOpen(event) {
+    this.connection.send(JSON.stringify(this.state.session));
```

Comment:
JSON.stringify on a string returns a string surrounded by quotes, why do we need the quotes?